### PR TITLE
Add try-catch around Cassandra bootstrap

### PR DESCRIPTION
**Pull request by acron0**

### CODE REVIEW

File: src/witan/workspace/components/cassandra.clj [hunk 1]
```diff
@@ -58,17 +58,22 @@
...
-    (log/info "Connecting to Cassandra..." host keyspace)
-    (let [conn (create-connection host keyspace)]
-      (assoc component :connection conn)))
+          (log/info "Connecting to Cassandra..." host keyspace)
+          (let [conn (create-connection host keyspace)]
+            (assoc component :connection conn)))
+      (catch Exception ex
+        (do
+          (log/error "Failed to start Cassandra:" ex)
+          component))))
```
Comment (Eleonore9) [inline src/witan/workspace/components/cassandra.clj:76]: I'm just curious. Why do you need to return the component when the starting fails?

Comment (acron0) [reply src/witan/workspace/components/cassandra.clj:76]: Just to be tidy, I think. I'm not sure what happens if you _don't_ return it

Comment (Eleonore9) [reply src/witan/workspace/components/cassandra.clj:76]: ok

**Comment by Eleonore9** [comment]
lgtm :shipit:

**Pull request merged**